var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -60,6 +60,10 @@ class Openblas(MakefilePackage):
     patch('openblas_icc_openmp.patch', when='%intel@16.0:')
     patch('openblas_icc_fortran.patch', when='%intel@16.0:')
 
+    # Fixes compilation error on POWER8 with GCC 7
+    # https://github.com/xianyi/OpenBLAS/pull/1098
+    patch('power8.patch', when='@0.2.18:0.2.19')
```

Comment:
I think it is necessary to add a `conflict` for older versions which don't have PowerPC support. According to https://github.com/xianyi/OpenBLAS/issues/1078 the asm code before patching was simply broken, so I would prefer patching affected versions whenever building on a PowerPC. The patch was created from an merged official PR so I would say it doesn't conflict with the claim of yours.